protostar/testing/test_collector.py

```diff
@@ -293,13 +294,13 @@ def _build_test_suite_from_test_suite_info(
 
     def _collect_test_cases(
         self,
-        function_names: List[str],
+        function_names: List[Tuple[str, AvailableGas]],
```

Comment:
Those aren't names anymore 🤔

Reply:
Also, you don't need to pass gas here, right?

Reply:
You dont' but pyright requires the same type here